Number_types/include/CGAL/NT_converter.h

```diff
@@ -72,6 +83,17 @@ struct NT_converter < double, double >
     }
 };
 
+template <>
+struct NT_converter < float, float >
```

Comment:
Is not there already `template <class NT1> struct NT_converter<NT1, NT1>`?!

Reply:
I just had a look: you are correct, but for some reason there is also a specialization for `double/double` (which is why I introduced the `float/float` one). Both could be removed, right?

Reply:
Yes I think so. To be checked.